dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/config/MasterDispatchTimeoutCheckerConfig.java

```diff
@@ -0,0 +1,42 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one or more
+ * contributor license agreements.  See the NOTICE file distributed with
+ * this work for additional information regarding copyright ownership.
+ * The ASF licenses this file to You under the Apache License, Version 2.0
+ * (the "License"); you may not use this file except in compliance with
+ * the License.  You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.apache.dolphinscheduler.server.master.config;
+
+import java.time.Duration;
+
+import lombok.Data;
+
+/**
+ * Configuration for the master's task dispatch timeout checker.
+ * If enabled, tasks that remain in the dispatch queue longer than {@link #maxTaskDispatchDuration} will be marked as failed to prevent indefinite queuing.
+ */
+@Data
+public class MasterDispatchTimeoutCheckerConfig {
+
+    /**
+     * Whether to enable the dispatch timeout checking mechanism.
+     */
+    private boolean enabled = false;
+
+    /**
+     * Maximum allowed time for a task to be dispatched to a worker.
+     * Tasks exceeding this duration in the dispatch queue will be failed.
+     * Examples: "2m", "5m", "30m". Defaults to 5 minutes.
+     */
+    private Duration maxTaskDispatchDuration = Duration.ofMinutes(5);
```

Comment:
```suggestion
    private Duration maxTaskDispatchDuration;
```
When we don't have a good suggested value, don't set a default value, as this can mislead many users.

Reply:
> When we don't have a good suggested value, don't set a default value, as this can mislead many users.

ok